### ISSUE TITLE
Flashing Function Causes Triple Flash on E-Ink Devices

### ISSUE THREAD

**Issue by HaowieLi** [mihonapp/mihon#1251]
### Steps to reproduce

1.	Open mihon on an E-Ink device.
2.	toggle to the next page.
3.	Observe the triple flash sequence 

### Expected behavior

The screen flash only once for a clean transition.

### Actual behavior

The screen flashes three times, showing the next page, then a white screen, and then the next page again.

### Mihon version

r6897

### Android version

harmonyos 2.1.0

### Device

huawei matepad paper

### Other details

This behavior seems to be caused by the way the page toggle is handled. When transitioning to the next page, there appears to be a very short delay during which the next page is displayed momentarily before the screen flashes white, and then the next page is shown once more. This triple flash issue disrupts the reading experience on E-Ink devices.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Smol-Ame** [comment]
Just checking, have you tried adjusting the Flash on page change settings to see if maybe you need to increase the duration or something?

![Screenshot_20240921_205629_Mihon.png](https://github.com/user-attachments/assets/3f0ababf-9149-41a0-98d0-1f3df8943183)

![Screenshot_20240921_205642_Mihon.png](https://github.com/user-attachments/assets/452d44d9-71f9-42ce-b1e5-87463892b050)

**Comment by HaowieLi** [comment]
> Just checking, have you tried adjusting the Flash on page change settings to see if maybe you need to increase the duration or something?
> 
> ![Screenshot_20240921_205629_Mihon.png](https://private-user-images.githubusercontent.com/155411819/369663498-3f0ababf-9149-41a0-98d0-1f3df8943183.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5NzgxMDEsIm5iZiI6MTcyNjk3NzgwMSwicGF0aCI6Ii8xNTU0MTE4MTkvMzY5NjYzNDk4LTNmMGFiYWJmLTkxNDktNDFhMC05OGQwLTFmM2RmODk0MzE4My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyMlQwNDAzMjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NjFjYWU0YzUxY2I4MDViMzJmYzI4OTk5NzNlZTg2NzU3MjhiODQ3MWZjYjFkMzE2YjdkNjYwYzM3ZWFlMzM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.eyMjO2pM_3d0217nrbJtDWZbW-0tGXum5Lz70FmZ5Bo)
> 
> ![Screenshot_20240921_205642_Mihon.png](https://private-user-images.githubusercontent.com/155411819/369663503-452d44d9-71f9-42ce-b1e5-87463892b050.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5NzgxMDEsIm5iZiI6MTcyNjk3NzgwMSwicGF0aCI6Ii8xNTU0MTE4MTkvMzY5NjYzNTAzLTQ1MmQ0NGQ5LTcxZjktNDJjZS1iMWU1LTg3NDYzODkyYjA1MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyMlQwNDAzMjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00Nzg5OGNjYmJmOGE5ZmExZTc3ZTJiN2NiMTMyNGEzNDI5YzI1NzFiY2Q0OTgwZGRmMWRlZjVmOWJmYjQ4ZDkxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.37CV1gV--7fKHETj0iGNQthnqBRjSr6jbClk0YmFebE)

I've tried all settings with no effect, and I found that only the white flash is effective in removing ghosting

**Comment by Smol-Ame** [comment]
Are you able to use another device to record the behavior on your Huawei Matepad Paper?

**Comment by HaowieLi** [comment]
> Are you able to use another device to record the behavior on your Huawei Matepad Paper?
> 
> If you are able to, and when you upload the video to here, make sure it's fully uploaded before you hit send/submit comment.

of course! Here it is

https://github.com/user-attachments/assets/0f42b847-8730-429f-872a-9173449717ba
